src/main/java/net/masterthought/cucumber/json/Features.java

```diff
@@ -0,0 +1,18 @@
+package net.masterthought.cucumber.json;
+
+import java.util.List;
+
+/**
+ * Created by suci on 11/16/17.
```

Comment:
same

please clean up the code before merge request

Reply:
done